File: Primeira lista pt2/Exercicio11.java
import java.util.Scanner;

public class Exercicio11 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Digite a largura: ");
        int largura = sc.nextInt();

        System.out.print("Digite a altura: ");
        int altura = sc.nextInt();

        for (int i = 0; i < altura; i++) {

            for (int j = 0; j < largura; j++) {
                System.out.print("* ");
            }

            System.out.println();
        }

        sc.close();
    }
}

File: lista pt2/Exercicio11.js
let largura = Number(
    prompt("Digite a largura do quadrado:")
);

let altura = Number(
    prompt("Digite a altura do quadrado:")
);

for (let i = 0; i < altura; i++) {

    let linha = "";

    for (let j = 0; j < largura; j++) {
        linha += "* ";
    }

    console.log(linha);
}